### PR TITLE
fixed #16847: updated font

### LINKED ISSUE #16847: Update font
### NetBox version

v4.1.0-dev

### Feature type

Change to existing functionality

### Proposed functionality

Update font used throughout app to match brand guidelines.

### Use case

Continuity.

### Database changes

_No response_

### External dependencies

_No response_

### PR DESCRIPTION
**Pull request by andrewgormley**
### Fixes: #16847

Updated font to Plus Jakarta Sans

### CODE REVIEW

**Comment by sleepinggenius2** [comment]
It is really bad practice to not define any fallback fonts. Instead, I would suggest using `--tblr-font-sans-serif: 'Plus Jakarta Sans', system-ui, sans-serif;` at the very least. Also, fonts are all currently served locally. It seems to be a regression to add a new dependency on an external CDN. We have contractors that use VDIs with very limited network access and they would not be able to load the font as currently defined, coupled with not having a fallback, the results would be indeterminate and undesirable from a UX standpoint.

**Review by arthanson — CHANGES_REQUESTED** [review]
As mentioned by sleepinggennius, the font needs to be local - NetBox must be able to run without outside internet connection.

File: netbox/project-static/styles/_variables.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Global variables
 
 // Set base fonts
-$font-family-sans-serif: 'Inter';
+$font-family-sans-serif: 'Plus Jakarta Sans Variable', 'Inter', system-ui, sans-serif;
```
Comment (arthanson) [inline netbox/project-static/styles/_variables.scss:4]: If we are using and including Plus Jakarta, do we still need Inter as a fallback?  And if not can we remove it from package.json as a dependency?

Comment (jeremystretch) [reply netbox/project-static/styles/_variables.scss:4]: We can remove it. Inter was selected as a custom font over whatever Tabler provides as a default IIRC. If we're using a different custom font, Inter can be removed.

**Review by jeremystretch — COMMENTED** [review]
I've noticed some odd spacing issues with the Jakarta font. For example, there is considerably less vertical space between the title and subtitle texts in the example below, even though element margin & padding remain unchanged. The subtitle text is also considerably larger in Jakarta, even though both are computed as 12px.

Original (Inter):
screenshot1

Modified (Jakarta):
screenshot2

File: netbox/project-static/styles/_variables.scss (hunk 1 above)
Comment (andrewgormley) [reply netbox/project-static/styles/_variables.scss]: Done.

**Comment by andrewgormley** [comment]
I believe the slight size different in the font faces is okay, but I have added some padding to the page header title to give it some breathing room.

**Review by jeremystretch — APPROVED** [review]

**Pull request merged**